Danger-Swift/Package.swift

```diff
@@ -13,12 +13,15 @@ let package = Package(
     ],
     dependencies: [
 //        .package(url: "https://github.com/danger/swift", from: "2.0.7")
-        .package(path: "/Users/antoinevanderlee/Documents/GIT-Projects/Eigen/swift")
+        .package(path: "/Users/antoinevanderlee/Documents/GIT-Projects/Eigen/swift"),
```

Comment:
The package declaration contains the commented out packages and reference to local file paths. I assume for testing purposes. Was this intended to be committed?

Reply:
It was part of WIP as it was not yet released in the original repo. It is now, so I've updated it!

Reply:
Ah gotta, you mentioned it in the description.